### PR TITLE
UX refactor dialog box

### PR DESCRIPTION
**Pull request by SL-SWoods**
Update confirmation dialog box to keep with new UX designs
image
- Remove border
- Change icons to words
- Larger buttons for selection options
- Centre dialog text accordingly. 

Unable to match specs on 
- Rounded border of the window. 
Need to confirm colours, as they do not appear to match the main colours being used within Stockflux.

### CODE REVIEW

File: packages/stockflux-components/src/components/popups/PopupWindow.js [hunk 1]
```diff
@@ -26,11 +26,10 @@ const PopupWindow = ({ message, options, children }) => {
       saveWindowState: false,
       showTaskbarIcon: false,
       resizable: false,
-      backgroundColor: '#28313D',
       waitForPageLoad: true,
       alwaysOnTop: true,
-      defaultWidth: 300,
-      defaultHeight: 100
+      defaultWidth: 270,
```
Comment (kangelSL) [inline packages/stockflux-components/src/components/popups/PopupWindow.js]: Should this be 275? The spec width looks like it's 50px + 50px + 175px = 275px

File: packages/stockflux-components/src/components/buttons/dialog-button/DialogButton.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  height: 30px;
+  width: 30px;
+  padding: 5px;
+  margin: 2px;
+}
+
+.dialog svg {
+  margin: auto;
+  display: block;
+  shape-rendering: auto;
```
Comment (kangelSL) [inline packages/stockflux-components/src/components/buttons/dialog-button/DialogButton.css]: If the default value is `auto`, is this needed?

File: packages/stockflux-launcher/public/popupWindow.css [hunk 3]
```diff
@@ -1,23 +1,40 @@
+html {
+    border-radius:20px;
```
[the remark this answers is not in the document]
Comment (SL-SWoods) [reply packages/stockflux-launcher/public/popupWindow.css]: should have been 5px, it was 20px when i was testing to see if radius was working correctly.

File: packages/stockflux-components/src/components/popups/PopupWindow.js (hunk 1 above)
Comment (SL-SWoods) [reply packages/stockflux-components/src/components/popups/PopupWindow.js]: the specs say 270x145, but if you look at the maths for the individual sections it adds up to 275. unsure which is the correct ones will confirm with UX.

File: packages/stockflux-components/src/components/buttons/dialog-button/DialogButton.css (hunk 2 above)
Comment (SL-SWoods) [reply packages/stockflux-components/src/components/buttons/dialog-button/DialogButton.css]: that entire section of css was not needed as we have no svgs within the dialogbuttons, so i have removed it.

File: packages/stockflux-components/src/components/popups/PopupWindow.js (hunk 1 above)
Comment (SL-SWoods) [reply packages/stockflux-components/src/components/popups/PopupWindow.js]: Confirmed the width should be 270.

**Pull request merged**